hw2/weather-api.py

```diff
@@ -0,0 +1,56 @@
+from requests import get
+import click
+
+BASE_URL = "http://api.weatherstack.com/current"
+
+
+def get_temperature_unit(temperature_name):
+    names_units = {"Celsius": "m", "Fahrenheit": "f"}
+    temperature_unit = None
+    try:
+        temperature_unit = names_units[temperature_name]
+    except KeyError:
+        print("The temperature name '{}' doesn't exist.".format(temperature_name))
+        keys_list = list(names_units.keys())
+        print("Please choose {0} or {1}.".format(keys_list[0], keys_list[1]))
+    return temperature_unit
+
+
+def get_temperature(token, city_name, units):
+    temperature = None
+    good_response_code = 200
+    request_failed_code = 615
```

Comment:
Not sure what happens with all the rest of the error codes which are defined here:
https://weatherstack.com/documentation#api_error_codes
Especially error codes 101, 104, 602

Reply:
When the passed city not found, api returns  request failed with error code 615. so i implement my own message when i get the code. I didn't found another way to verify the city.
With the fix i added, there is output of a code and a message.